examples/chat-tokio.rs

```diff
@@ -53,7 +53,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     // Create a random PeerId
     let id_keys = identity::Keypair::generate_ed25519();
     let peer_id = PeerId::from(id_keys.public());
-    println!("Local peer id: {:?}", peer_id);
```

Comment:
main `cargo.toml` [`rust-version` is `1.60`](https://github.com/libp2p/rust-libp2p/blob/master/Cargo.toml#L4) which was done on https://github.com/libp2p/rust-libp2p/pull/2646 while all other crates are still with `1.56`. Should we also upgrade the other crates for consistency?  @mxinden @thomaseizinger 
In this case captured identifiers were only [introduced in `1.58`](https://blog.rust-lang.org/2022/01/13/Rust-1.58.0.html#captured-identifiers-in-format-strings), I wonder if clippy takes attention to that and there are more cases like this not being linted.

Reply:
right, just noticed https://github.com/libp2p/rust-libp2p/pull/3090#pullrequestreview-1170198498, thanks Thomas!

Reply:
Yes there are more cases, I can change them as well. I discovered this with clippy::pedantic. I didn't have time to look through all things clippy::pedantic highlighted, that is beyond the scope of this PR and is also a lot of work.

Reply:
We ignore `clippy::pedantic` in here (see `custom-clippy` alias in `.cargo/config.toml`)